presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreenState.kt

```diff
@@ -23,6 +23,6 @@ data class EventScreenState(
     val endDate: ZonedDateTime = ZonedDateTime.now(ZoneId.systemDefault()),
     val isStartDateTime: Boolean = false,
     val shouldOpenDropdown: Boolean = false,
-    val alarmItem: AlarmItem = AlarmItem(ZonedDateTime.now(), null),
-    @StringRes val alarmReminderText: Int = me.androidbox.component.R.string.ten_minutes_before,
+    @StringRes val alarmReminderText: Int = R.string.ten_minutes_before,
```

Comment:
Do you really need this?

One option could be to have the AlarmReminderItem receive the stringRes as parameter
So you don't do it here in the State

Mostly so the Enum takes care of everything about itself